soda/src/main/java/it/okkam/opendata/models/soda/SodaResult.java

```diff
@@ -6,6 +6,7 @@
   SodaMetadata metadata;
   String permalink;
   String link;
+  String preview_image_url;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "preview_image_url" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3AS00116)